### DISCUSSION TITLE
Extensibility/plugins for adding new sources of tasks

### DISCUSSION THREAD

**Discussion by HammyHavoc** [alainm23/planify#1250] [Ideas]
It would be awesome if there was a plugin/extension system for writing support for other systems, e.g., I use Active Collab, OpenProject, Vikunja etc.

**Comment by bryandesrosiers** [comment]
That would solve a lot of "tasks in various places" issues to organize and action on them.

**Reply by HammyHavoc** [reply]
Yes, it's so much friction to juggle all this stuff. After consolidating all my messaging apps into one via Matrix bridges into Element, I want to do the same for tasks.
